cmd/transform/transform.go

```diff
@@ -150,6 +150,9 @@ func (o *Options) run() error {
 	}
 	if len(o.OptionalFlags) > 0 {
 		runner.OptionalFlags = optionalFlagsToLower(o.OptionalFlags)
+		for k, v := range runner.OptionalFlags {
+			log.Infof("optionalarg k: %v, v: %v", k, v)
+		}
```

Comment:
I think this could simply be:
```suggestion
		lob.Debugf("parsed optional-fags: %v", runner.OptionalFlags)
```

was playing around with it and this is what it would look like:
```
$ ./crane transform --export-dir=export --transform-dir=transform --optional-flags="add-annotations=migration-using-crane=true,foo=bar,baz=zoo" --debug
DEBU[0000] optional-flags: map[add-annotations:migration-using-crane=true baz:zoo foo:bar]
```

Reply:
Actually, that was debug code that I had intended to remove. But I could keep it as `Debugf` as you suggest. Will update this with the change.

Reply:
updated